librocksdb_sys/crocksdb/c.cc

```diff
@@ -6506,11 +6501,6 @@ void ctitandb_options_set_discardable_ratio(ctitandb_options_t* options,
   options->rep.blob_file_discardable_ratio = ratio;
 }
 
-void ctitandb_options_set_sample_ratio(ctitandb_options_t* options,
```

Comment:
It seems that the current TiKV is still using [this](https://github.com/tikv/tikv/blob/master/src/config.rs#L178 ) ? Removing this function will cause a link error in TiKV. @waynexia @tabokie

Reply:
```
  = note: /usr/bin/ld: /workspace/pingcap/tikv/target/x86_64-unknown-linux-gnu/release/deps/tikv_server-d9f913b11da850d2.tikv-6379e790445eb3a2.tikv.4fa2f1af-cgu.0.rcgu.o.rcgu.o: in function `tikv::config::TitanCfConfig::build_opts':
          tikv.4fa2f1af-cgu.0:(.text._ZN4tikv6config13TitanCfConfig10build_opts17h9088003f9f62e419E+0x8b): undefined reference to `ctitandb_options_set_sample_ratio'
          clang: error: linker command failed with exit code 1 (use -v to see invocation)

  = help: some `extern` functions couldn't be found; some native libraries may need to be installed or have their path specified
  = note: use the `-l` flag to specify native libraries to link
  = note: use the `cargo:rustc-link-lib` directive to specify the native libraries to link with Cargo (see https://doc.rust-lang.org/cargo/reference/build-scripts.html#cargorustc-link-libkindname)

error: could not compile `tikv-server` due to previous error
warning: build failed, waiting for other jobs to finish...
error: linking with `clang` failed: exit status: 1
```

Reply:
It's removed in https://github.com/tikv/titan/pull/251. TiKV end needs to deprecate this option.